src/selfplay/game.cc

```diff
@@ -25,10 +25,12 @@ namespace lczero {
 
 namespace{
 const char* kReuseTreeeStr = "Reuse the node statistics between moves";
```

Comment:
Maybe while you here, also change Treee to Tree? I missed it in previous review.

Reply:
Done